src/Http/Authentication.Abstractions/src/IAuthenticationResultFeature.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Authentication
+{
+    /// <summary>
+    /// Used to store the AuthenticateResult for the HttpContext.
+    /// </summary>
+    public interface IAuthenticationResultFeature
```

Comment:
Should maybe name this something a little less crappy...  DefaultAuthenticateResultFeature since it maps to DefaultAuthenticate?